src/ExactOnline.Client.Sdk/Controllers/ExactOnlineClient.cs

```diff
@@ -23,8 +25,6 @@ public class ExactOnlineClient
 
         public int Division { get; private set; }
 
-        public EolResponseHeader EolResponseHeader { get; internal set; }

```

Comment:
Better to keep the EolResponseHeader exposed here. I see you have moved it to the ApiConnector, but that one isn't exposed anywhere? Also to not make a breaking change...